src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,33 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) throws Exception {
+
+        SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+        Cipher cipher = Cipher.getInstance("AES");
+        cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+        byte[] encrypted = cipher.doFinal(value.getBytes());
+        return StringHelper.byteArrayToHexString(encrypted);
+
+    }
+
+    @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        String plaintextLength28 = "AAAAAAA";
+        //Using plaintext > 128 bits to check any differences operations.
```

Comment:
It's not clear to which line this comment applies to. If it applies to the next line only, there should be a blank line after that. Otherwise it applies to the entire block of code that follows.
```
// Comment about statement 1
statement1;

statement2;
```

On a related note, can the comment be a bit more informative?